src/main/interface/cli.c

```diff
@@ -3575,24 +3575,17 @@ STATIC_UNIT_TESTED void cliSet(char *cmdline)
     }
 }
 
+#ifdef SIMULATOR_BUILD
```

Comment:
Not sure this should be in `cli.c`, there might be other places wanting to use `mcuTypeName` (CMS, MSP, ...). Should we change this to a define and stick it somewhere like `platform.h` or `target/common_post.h`?